api/lib/domain/usecases/get-certification-center-invitation.js

```diff
@@ -0,0 +1,27 @@
+const {
+  AlreadyExistingCertificationCenterInvitationError,
+  CancelledCertificationCenterInvitationError,
+} = require('../../domain/errors');
+
+module.exports = async function getCertificationCenterInvitation({
+  certificationCenterInvitationId,
+  certificationCenterInvitationCode,
+  certificationCenterInvitationRepository,
+}) {
+  const foundCertificationCenterInvitation = await certificationCenterInvitationRepository.getByIdAndCode({
+    id: certificationCenterInvitationId,
+    code: certificationCenterInvitationCode,
+  });
+
+  if (foundCertificationCenterInvitation.isCancelled) {
+    throw new CancelledCertificationCenterInvitationError('Invitation was cancelled');
```

Comment:
One day it could be cool to choose between errors in french and les erreurs en anglais ^^

Reply:
J'avoue m'être basée sur le fichier orga, du coup j'imagine que c'est mieux de renvoyer toutes les errors en français